tools/c7n_mailer/c7n_mailer/azure/deploy.py

```diff
@@ -41,7 +41,7 @@ def build_function_package(config, function_name):
 
     package.build(None,
                   modules=['c7n', 'c7n-azure', 'c7n-mailer', 'applicationinsights'],
-                  non_binary_packages=['pyyaml', 'pycparser', 'tabulate',
+                  non_binary_packages=['pyyaml', 'pycparser', 'tabulate', 'jmespath',
```

Comment:
Would it be easier to reference the pip package through `requirements.txt`? https://pypi.org/project/jmespath/

Reply:
this really isn't about requirements for install (re requirements.txt), its about packaging dependencies for upload, in that context at least for azure needs distinctions between binary and source assets. also requirements.txt assumes git clone/source installs not package/pip installs for which that file doesn't convey.